README.md

```diff
@@ -97,6 +97,46 @@ end
 
 Note how we now need to add the line `using MyPackage, Test` manually to our `@testitem` so that we have access to the `foo` function and `@test` macro.
 
+##### Shared modules
+
+You can reuse code by creating modules with `@testsetup`:
+
+```julia
+@testsetup module TestSetup
+    const x = 10
+    getfloat() = rand()
+end
+```
+
```

Comment:
Does this definition have to occur in a specific place in a package, which should be mentioned here?  Or will it be picked up anywhere in the codebase?  I'm asking because with `TestItemRunner v0.2.3`, I can't seem to get a `TestSetup` created that's recognized within my `@testitem` sections (I put the `@testsetup` section in my `runtests.jl`).